framework/src/index.ts

```diff
@@ -16,6 +16,7 @@ export { Application } from './application';
 export { version } from './version';
 export { systemDirs } from './application/system_dirs';
 export { BasePlugin, PluginInfo } from './plugins/base_plugin';
+export { IPCChannel } from './controller/channels';
 export type { BaseChannel } from './controller/channels';
 export type { EventsArray } from './controller/event';
```

Comment:
maybe expose these from `controller/index.ts` and `export {} from './controller'`

Reply:
These are exposed at `controller/index.ts` but we need only `IPCChannel` channel to expose from framework.